app/controllers/accounts/registrations_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class Accounts::RegistrationsController < Devise::RegistrationsController
+  # before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  GET /resource/sign_up
+  def new
+    super
+  end
+
+  # POST /resource
+  def create
+    debugger
+    super
+  end
+
+  # GET /resource/edit
+  # def edit
+  #   super
+  # end
+
+  # PUT /resource
+  # def update
+  #   super
+  # end
+
+  # DELETE /resource
+  # def destroy
+  #   super
+  # end
+
+  # GET /resource/cancel
+  # Forces the session data which is usually expired after sign
+  # in to be expired now. This is useful if the user wants to
+  # cancel oauth signing in/up in the middle of the process,
+  # removing all OAuth session data.
+  # def cancel
+  #   super
+  # end
+
+  # protected
+
+  # If you have extra params to permit, append them to the sanitizer.
+  # def configure_sign_up_params
+  #   devise_parameter_sanitizer.permit(:sign_up, keys: [:attribute])
```

Comment:
Why do we have so much commented code?

Reply:
All the files under controller/accounts/ are generated by devise , these files are provided in order to add any extra functionality to devise SignIn/SignUp Flow. These files stays commented unless we want to add any user desired functionality in these controllers as they are inherited from the Devise Controllers.
These controllers were generated using
rails generate devise:controllers accounts